Makefile

```diff
@@ -13,11 +13,23 @@ endif
 BUILD_DATE ?= $(shell date -u '+%Y-%m-%d-%H:%M UTC')
 VERSION_FLAGS := -ldflags='-X "main.buildVersion=$(BUILD_VERSION)" -X "main.buildTime=$(BUILD_DATE)"'
 
+VERSION ?= 1.1.9
```

Comment:
The latest HPK version is 0.1.2... I suggest you lower this to 0.2.1 and probably load it from a global file. again, check our other repos